File: Dataset/Leetcode/train/26/90.java
 class Solution {
    public int XXX(int[] nums) {
        int left=0,right=0;
        int len=nums.length;
        while(right<len){
            while(right<len&&nums[right]==nums[left]){
                right++;
            }
            if(right<len&&nums[right]!=nums[left]){
                nums[++left]=nums[right];
            }
        }
        return left+1;
    }
}

File: Dataset/Leetcode/test/22/90.cpp
 class Solution {
    int maxleft;
    int maxright;
    vector<string>ans;
public:
    void set(string str,int left,int right)
    {
        //cout<<str<<endl;
        if(left+right==maxleft+maxright)
        {
            ans.push_back(str);
            return ;
        }
        if(left<maxleft)
            set(str+"(",left+1,right);
        if(right<left&&right<maxright)
            set(str+")",left,right+1);
    }

    vector<string> XXX(int n) 
    {
        if(n==0)return  ans;
        maxleft = n;
        maxright = n;
        set("",0,0);
        return ans;
    }
};

